l10n_br_base/models/res_partner.py

```diff
@@ -0,0 +1,220 @@
+# Copyright (C) 2009 Gabriel C. Stabel
+# Copyright (C) 2009 Renato Lima (Akretion)
+# Copyright (C) 2012 Raphaël Valyi (Akretion)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from erpbrasil.base.fiscal import cnpj_cpf, ie
+    from erpbrasil.base import misc
+except ImportError:
+    _logger.error("Biblioteca erpbrasil.base não instalada")
+
+
+class Partner(models.Model):
```

Comment:
I don't see reason to change file formation, should just change import order:

https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_base/models/res_partner.py